src/libstd/sys_common/backtrace.rs

```diff
@@ -25,45 +27,107 @@ pub const HEX_WIDTH: usize = 18;
 #[cfg(target_pointer_width = "32")]
 pub const HEX_WIDTH: usize = 10;
 
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+pub enum PrintFormat {
+    Full = 2,
```

Comment:
OK. In fact, it controls if the unwinding should use the short or long backtrace format.